medusa/metadata/kodi_12plus.py

```diff
@@ -214,7 +214,7 @@ def _ep_data(self, ep_obj):
         """
         eps_to_write = [ep_obj] + ep_obj.related_episodes
 
-        series_obj = self._get_show_data(ep_obj.series)
+        series_obj = self._get_show_data(ep_obj.series, ep_obj)
```

Comment:
kodi12 is the only one that creates episode thumbnails, but in the future, we should pass the ep_obj if we want to get episode related show data.